### PR TITLE
EZP-30312: Performing actions from subitems list does not refresh the Content Tree

### PR DESCRIPTION
**Pull request by tischsoic**
JIRA: https://jira.ez.no/browse/EZP-30312

Related PR from admin-ui for location visibility change: https://github.com/ezsystems/ezplatform-admin-ui/pull/921

### CODE REVIEW

File: src/modules/content-tree/content.tree.module.js [hunk 1]
```diff
@@ -43,6 +44,8 @@ export default class ContentTreeModule extends Component {
         loadSubtree(this.props.restInfo, this.subtree, (loadedSubtree) => {
             this.setInitialItemsState(loadedSubtree[0]);
         });
+
+        document.body.addEventListener('ctRefresh', this.refreshContentTree, false);
```
Comment (tischsoic) [inline src/modules/content-tree/content.tree.module.js]: I've chosen `ctRefresh` based on names such as: `pbIframeLoaded`, or` fbFormBuilderUnloaded`, but I don't think it is perfect. What do you think @dew326 ?

**Comment by tischsoic** [comment]
As agreed with @SylvainGuittard, Content Tree will refresh on bulk delete/move and location visibility change.

File: src/modules/content-tree/content.tree.module.js (hunk 1 above)
Comment (dew326) [reply src/modules/content-tree/content.tree.module.js]: I think we should prefix events with `ez-` like we have with `ez-notify` or `ez-bookmark-change` so for me the name should be `ez-content-tree-refresh`

File: src/modules/content-tree/content.tree.module.js [hunk 2]
```diff
@@ -43,6 +44,8 @@ export default class ContentTreeModule extends Component {
         loadSubtree(this.props.restInfo, this.subtree, (loadedSubtree) => {
             this.setInitialItemsState(loadedSubtree[0]);
         });
+
+        document.body.addEventListener('ez-content-tree-refresh', this.refreshContentTree, false);
```
Comment (dew326) [inline src/modules/content-tree/content.tree.module.js]: This `componentDidMount()` has the early return. There is a possibility this event will not be attached. Seems like should be before the return or am I missing something?

Comment (tischsoic) [reply src/modules/content-tree/content.tree.module.js]: You're right, good catch. :)

**Review by dew326 — APPROVED** [review]

**Review by katarzynazawada — APPROVED** [review]

**Pull request merged**